lms/templates/courseware/course_navigation.html

```diff
@@ -83,27 +78,11 @@
 % if disable_tabs is UNDEFINED or not disable_tabs:
 <nav class="${active_page} wrapper-course-material" aria-label="${_('Course Material')}">
   <div class="course-material">
-    <ol class="course-tabs">
-      % for tab in get_course_tab_list(request, course):
-        <%
-            tab_is_active = (tab.tab_id == active_page) or (tab.tab_id == default_tab)
-        %>
-          <li>
-             <a href="${tab.link_func(course, reverse) | h}" class="${url_class(tab_is_active)}">
-                 ${_(tab.name) | h}
-                 % if tab_is_active:
-                   <span class="sr">, current location</span>
-                 %endif
-                 % if tab_image:
-                   ## Translators: 'needs attention' is an alternative string for the
-                   ## notification image that indicates the tab "needs attention".
-                   <img src="${tab_image}" alt="${_('needs attention')}" />
-                 %endif
-             </a>
-          </li>
-      % endfor
-      <%block name="extratabs" />
-    </ol>
+     <%
+      tab_list = get_course_tab_list(request, course)
+      %>
+     <%static:optional_include_mako file="courseware/tabs.html" is_theming_enabled="True"
```

Comment:
@StraNNiKK I'm wondering what could be the reason for using `optional_include_mako` for including course tabs instead of built-in include since course tabs are not optional part of courseware page?.